examples/src/read.c

```diff
@@ -232,8 +245,11 @@ int main(int argc, char* argv[])
                     "Number of processes:       %d\n"
                     "Each process wrote:        %.2lf MiB\n"
                     "Total data written:        %.2lf MiB\n"
+                    "File stat time:            %.6lf sec\n"
+                    "File open time:            %.6lf sec\n"
                     "Maximum read time:         %.6lf sec\n"
                     "Maximum check time:        %.6lf sec\n"
+                    "File close time:           %.6lf sec\n"
                     "Aggregate read bandwidth:  %.3lf MiB/s\n"
                     "Effective read bandwidth:  %.3lf MiB/s\n",
```

Comment:
3 new lines get added to the output. The read CI tests will need to be updated:
Here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/120-read-tests.sh#L71
and here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/120-read-tests.sh#L88